README.md

```diff
@@ -47,7 +47,7 @@ metadata:
     k8s-sidecar-target-directory: "/path/to/target/directory"
 ```
 
-If the filename ends with `.url` suffix, the content will be processed as a URL which the target file contents will be downloaded from.
+If the filename ends with `.url` suffix, the content will be processed as a URL which the target file contents will be downloaded from. If the source requires authentication, currently http basic-auth and header based token authentication are supported. Please see the variables below for details. 
```

Comment:
here it's described as header based token auth but further down it's described as parameters and if I read the impl correctly it's actually doing query parameters, right?

Reply:
Fixed accordingly.